colibri/src/icons.rs

```diff
@@ -16,6 +22,13 @@ pub enum FileTypeError {
     UnsupportedFileType,
 }
 
+sol! {
+    #[sol(rpc)]
+    interface IERC721Metadata {
+        function tokenURI(uint256 tokenId) external view returns (string memory);
+    }
+}
```

Comment:
should we have them in a contract file? I believe having them disperse across the code is not a good idea